### ISSUE TITLE
Feature: Don't show again for Quick Save

### ISSUE THREAD

**Issue by Omegaplexx** [localsend/localsend#1684]
### Is your feature request related to a problem? Please describe.

Hello! It's annoying to see that Quick Save dialog every time.
![Screenshot_20240821-235006_LocalSend~01~01](https://github.com/user-attachments/assets/ab08278e-216f-46c7-86e5-0e76998dfe7a)

### Describe the solution you'd like

I know it's useful, so I suggest the following solution: never show it on main screen (or only once) and always show it when enabled from Settings screen.

### Describe alternatives you've considered

Add a "Don't show again" checkbox for Quick Save dialog from main screen; always show it when enabled from Settings screen.

### Additional context

Sorry if there's any mistakes in my English. Thank you very much for LocalSend, it helps me a lot!

**Comment by sergd88** [comment]
@Tienisto No, please, no. It’s not a very good idea. It’s useful on the main screen. I personally know enough people who use this regularly. Even I, occasionally, use it due to the lack of "trusted devices". And having to go into settings each time to enable the option will clearly be inconvenient.

When it's used, for example, in a company of people you trust, where receiving requests would only be a nuisance. When you're transferring a large number of files not in a single transaction, because they are scattered across different disks and it’s easier to send them in parts.

As for the option to disable it, that would also be quite strange. Then you would need to add it for every element that, theoretically, can be hidden because you don’t like it.

Regarding different aesthetic tastes, let’s look at it from a practicality perspective. It’s very hard to accidentally physically hit this button, as it is located at the very bottom. Also, it somewhat complements the "Send" page by making it look less empty, as there aren’t many elements on it anyway.

**Comment by Tienisto** [comment]
> never show it on main screen (or only once) and always show it when being enabled from Settings screen.

I believe "it" refers to the informational popup when this setting is enabled

**Comment by sergd88** [comment]
> > never show it on main screen (or only once) and always show it when being enabled from Settings screen.
> 
> I believe "it" refers to the informational popup when this setting is enabled

Oops, sorry, I didn't read that carefully.
